### PR TITLE
Fix core examples so they don't panic

### PR DESCRIPTION
**Pull request by marsella**
The `basic` and `async` examples did not correctly encode the expected response. Now they do!

### CODE REVIEW

**Review by niklasad1 — APPROVED** [review]

**Comment by marsella** [comment]
Can I get a second review? Not sure if @tomusdrw is the best person - please tag someone else if appropriate.

**Review by tomusdrw — APPROVED** [review]

**Comment by tomusdrw** [comment]
Thanks!

**Pull request merged**
